src/lib/openables.ts

```diff
@@ -307,6 +308,14 @@ const osjsOpenables: UnifiedOpenable[] = [
 		output: Openables.NestBoxSeeds.table,
 		allItems: Openables.NestBoxSeeds.table.allItems
 	},
+	{
+		name: 'Bird nest (ent)',
+		id: 5070,
+		openedItem: getOSItem(5070),
+		aliases: ['bird nest (ent)', 'ent nest', 'ent bird nest', 'bird nest ent'],
+		output: nestTable,
+		allItems: nestTable.allItems
+	},
```

Comment:
Why is this dropping the red egg nest, and being called an ent nest?

Reply:
The ent red bird nest never got removed when majority of the other bird egg nests got removed in favour of giving both egg and nest separately. 
I would assume this is due to thinking ent nests are intended.